frb/surveys/sdss.py

```diff
@@ -93,7 +94,7 @@ def get_catalog(self, photoobj_fields=None, timeout=None, print_query=False):
         query += "p.objid, "
 
         query += "pz.z as redshift, pz.zErr as redshift_error\n"
-        query += "FROM PhotoObjAll as p\n"
+        query += "FROM PhotoObj as p\n"
```

Comment:
Can you remind me of the difference between `PhotoObjAll` and `PhotoObj` ?

Reply:
PhotoObjAll contains all detections and will often have multiple entries corresponding to the same object. PhotoObj is the cleaned version of this table without these redundancies.